lib/bookstore-scrapper/helpers/getTextbookInformation.ts

```diff
@@ -0,0 +1,180 @@
+// Return textbook information given link and it's parameters
+import { axiosRequest } from "./axiosRequest";
+import { randomUUID } from "crypto";
```

Comment:
What format of UUID does this generate? I think for other parts of the code, we were using:
```typescript
import { v4 } from "uuid";
```
Then using `v4()` to generate UUIDs

Reply:
Uses v4 format ([see doc](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID)). I saw it being used [in this file](https://github.com/calebmech/Book-Bazar/blob/00b6465cfd58d48ef1ccd58b6d581aefae00e3f4/lib/helpers/backend/tokens.ts) to generate UUID's and that's why I used it.

Will change it to use v4.